src/lm/arpa-file-parser.h

```diff
@@ -100,24 +131,28 @@ class ArpaFileParser {
   /// Pure override that must be implemented to process current n-gram. The
   /// n-grams are sent in the file order, which guarantees that all
   /// (k-1)-grams are processed before the first k-gram is.
-  virtual void ConsumeNGram(const NGram&) = 0;
```

Comment:
OK, it's fine then.

On Mon, Apr 4, 2016 at 12:02 AM, Kirill Katsnelson <notifications@github.com
> wrote:

> In src/lm/arpa-file-parser.h
> <https://github.com/kaldi-asr/kaldi/pull/649#discussion_r58326729>:
>
> > @@ -100,24 +131,28 @@ class ArpaFileParser {
> >    /// Pure override that must be implemented to process current n-gram. The
> >    /// n-grams are sent in the file order, which guarantees that all
> >    /// (k-1)-grams are processed before the first k-gram is.
> > -  virtual void ConsumeNGram(const NGram&) = 0;
>
> It's not quite precise
> <https://google.github.io/styleguide/cppguide.html#Type_Names> on that:
>
> Type names start with a capital letter and have a capital letter for each
> new word, with no underscores.
>
> So, strictly speaking, this depends on whether you treat "n-gram" as one
> word or two. :)
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/649/files/939259c797b78d91abfcfe582813d8cd3afccbf8#r58326729>
>